enterprise/api/pagination.py

```diff
@@ -21,7 +21,8 @@ def get_paginated_response(data, request):
     Returns:
         (Response): DRF response object containing pagination links.
     """
-    url = request.build_absolute_uri().rstrip('/') + '/'
+    url = urlparse(request.build_absolute_uri())._replace(query=None).geturl()
```

Comment:
@brittneyexline Turns out we aren't actually throwing anything away here. This line is meant to extract the protocol, host, and path from the request to the Enterprise service so that they can be used to update the next/previous URLs that are returned from the Catalog service. It is appropriate to use the query parameters from the next/previous URLs from the Catalog data below.